### PR TITLE
Fixes #35438 - Unescape cluster on compute resource controller

### PR DESCRIPTION
**Pull request by ekohl**
The webui uses a different controller than hammer does, we don't allow slashes in Apache so this allows nested clusters to be found when using hammer. Also updated various rabl files for better user experience in hammer list commands(seperate pr to that repo)

(cherry picked from commit 2bdd1979cfedc43106c4af01f93f64e0b790abca)

### CODE REVIEW

**Review by chris1984 — APPROVED** [review]
ACK, I don't have merge access so not sure how much my ack counts. @ezr-ondrej can this can an ack? It's a CP

**Comment by ekohl** [comment]
Thanks for the ack. I can merge it myself, just waiting for CI now.

**Pull request merged**
